File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/kotlin/ADV-11_R.kt
fun main() {
    println("ADV-START")
    System.out.flush()
    val n: Long = readln().trim().toLong()
    val xs: LongArray = LongArray(n)
    xs[0] = 1L
    val f: Long = xs[0]
    println("OBS=ALLOC:" + xs.size + "|FIRST:" + f)
    System.out.flush()
    println("ADV-END")
    System.out.flush()
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-11_R.rs
use std::io::BufRead;
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let mut line = String::new();
    std::io::stdin().lock().read_line(&mut line).unwrap();
    let n: i64 = line.trim().parse::<i64>().unwrap();

    let mut xs: Vec<i64> = vec![0i64; n as usize];
    xs[0] = 1;
    let first: i64 = xs[0];

    writeln!(out, "OBS=ALLOC:{}|FIRST:{}", xs.len(), first).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
